modules/javafx.graphics/src/main/native-glass/mac/GlassWindow+Java.m

```diff
@@ -94,10 +94,14 @@ - (void)_sendJavaWindowResizeEvent:(int)type forFrame:(NSRect)frame
     if (self->suppressWindowResizeEvent == NO)
     {
         GET_MAIN_JENV;
-        (*env)->CallVoidMethod(env, jWindow, jWindowNotifyResize,
-            [self->nsWindow isZoomed]
-                ? com_sun_glass_events_WindowEvent_MAXIMIZE
-                : type,
+
+        if([self->nsWindow isMiniaturized]) {
+            type = com_sun_glass_events_WindowEvent_MINIMIZE;
+        } else if([self->nsWindow isZoomed]) {
```

Comment:
Minor: can you add a space between `if` and `(` to match our coding style?

Reply:
Done!